README.md

```diff
@@ -1,2 +1,70 @@
 # k8s_crd
```

Comment:
What `_crd` means in this context, CRD or CoreDNS? 
What do you think about `k8s_dnsendpoint` as the name, to better reflect the usage of plugin?

Reply:
CRD, I had discussion about it with @ytsarev we agreed to name it as CRD. Project had initial name k8s_dnsendpoint (or k8s_extdns)